gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/resource/api/ApiPlansResource.java

```diff
@@ -124,55 +127,58 @@ public Response createApiPlan(@Valid @NotNull NewPlanEntity newPlanEntity) {
 
         PlanEntity planEntity = planService.create(GraviteeContext.getExecutionContext(), newPlanEntity);
 
-        return Response.created(this.getLocationHeader(planEntity.getId())).entity(PlanMapper.INSTANCE.convert(planEntity)).build();
+        return Response.created(this.getLocationHeader(planEntity.getId())).entity(planMapper.convert(planEntity)).build();
     }
 
     @GET
     @Path("/{plan}")
```

Comment:
The path param should follow the same format as others, and be suffixed by `Id`
```suggestion
    @Path("/{planId}")
```